src/C1T2X_OBU.py

```diff
@@ -169,6 +186,20 @@ def LAN_listening_thread():
 			if pkt:
 				if not parseLANPacket:
 					sendVANET(pkt[0])
+					# Wait for ack
+					waiting_for_ack = True
+					c1t2x_logger.info("Message sent, waiting for ack")
+					time.sleep(1.0)
```

Comment:
What is the purpose of this sleep?

Reply:
The purpose is to wait for 1 second before checking to see if we have received an ack or if we need to rebroadcast the message. This way we only retry every 1 second (if necessary) instead of firing off a bunch of messages.